code/modules/mob/living/carbon/xenomorph/strains/castes/boiler/trapper.dm

```diff
@@ -217,26 +212,33 @@
 	return ..()
 
 /datum/ammo/xeno/acid_shotgun
-	name = "acid ball"
+	name = "acid spatter"
 	ping = null
 	flags_ammo_behavior = AMMO_SKIPS_ALIENS|AMMO_STOPPED_BY_COVER|AMMO_IGNORE_ARMOR|AMMO_ACIDIC
 	bonus_projectiles_type = /datum/ammo/xeno/acid_shotgun/spread
 	hit_effect_color = "#FF0000"
 
 /datum/ammo/xeno/acid_shotgun/New()
 	..()
-	damage = 10
+	damage = 6
 	max_range = 5
 	damage_type = BURN
 	accuracy = HIT_ACCURACY_TIER_8
 	accuracy_var_low = PROJECTILE_VARIANCE_TIER_6
 	accuracy_var_high = PROJECTILE_VARIANCE_TIER_6
 	bonus_projectiles_amount = EXTRA_PROJECTILES_TIER_9
-	shrapnel_type = /datum/ammo/xeno/acid_shotgun/spread
-	shrapnel_chance = 100
+	bonus_projectiles_type = /datum/ammo/xeno/acid_shotgun/spread
+
+/datum/ammo/xeno/acid_shotgun/on_hit_mob(mob/living/carbon/target, obj/projectile/proj)
+	. = ..()
+	var/mob/living/carbon/xenomorph/xeno = proj.firer
+	var/datum/behavior_delegate/boiler_trapper/be_del = xeno.behavior_delegate
```

Comment:
can just be behavior i dont think you need to use snake case here, a small nitpick regardless